python/README.md

```diff
@@ -65,7 +65,10 @@ More extensive documentation (generated with Sphinx) is available in the `python
 ## Changelog
 
 - 2015-12-10 First public release (0.1)
-- 2016-08-16 Minor release:
-   1. the official Spark target is Spark 0.2
+- 2016-08-16 Minor release (0.2.0):
+   1. the official Spark target is Spark 2.0
    2. support for keyed models
+- 2017-09-14 Minor release (0.2.2):
+   1. The official Spark target is Spark >= 2.1
```

Comment:
Do we want to update travis to also test Spark 2.2?

Reply:
Oops, yes we should! Thanks for the catch :)

I had added Spark 2.2 to the build in #60 but forgot to merge that commit back into the feature branch (spark-2.2-patch) for this PR. I guess since I didn't change .travis.yml in this PR Github didn't detect any conflicts etc.